jwala-services/src/test/java/com/cerner/jwala/service/webserver/impl/WebServerServiceImplTest.java

```diff
@@ -132,7 +136,7 @@ public void setUp() throws IOException {
         mockWebServers11.add(mockWebServer);
         mockWebServers12.add(mockWebServer2);
 
-        wsService = new WebServerServiceImpl(webServerPersistenceService, resourceService, inMemService, StringUtils.EMPTY, binaryDistributionLockManager);
+        wsService = new WebServerServiceImpl(webServerPersistenceService, resourceService, inMemService, StringUtils.EMPTY, binaryDistributionLockManager, jvmPersistenceService);
```

Comment:
You made this class compile, but where is the junit test that actually tests that trying to add a dupe will throw an error?